lago/providers/libvirt/network.py

```diff
@@ -227,24 +228,48 @@ def make_ipv4(last):
                     )
                 )
 
-        if self.is_management():
-            domain_xml = ET.Element(
-                'domain', name=self._spec['dns_domain_name'], localOnly='yes'
-            )
-            net_xml.append(domain_xml)
-            net_xml.append(
-                self._generate_main_dns(
-                    self._spec['dns_records'], self._spec['dns_domain_name'],
-                    subnet
+        if utils.ver_cmp(self.compat, '0.36.11') >= 0:
+            if self.is_management():
+                domain_xml = ET.Element(
+                    'domain',
+                    name=self._spec['dns_domain_name'],
+                    localOnly='yes'
                 )
-            )
+                net_xml.append(domain_xml)
+                net_xml.append(
+                    self._generate_main_dns(self._spec['dns_records'], subnet)
```

Comment:
I think what makes sense is:
For all management networks - don't forward unqualified names, as we know this is the last endpoint and we probably don't want any unqualified DNS records outside the virtual network.
For all none-mgmt networks - forward(as they don't have DNS).

Prior to 6386bc8 we had forwardPlainNames='yes' in all networks(hardcoded in the template).